nifi-commons/nifi-security-utils/src/test/groovy/org/apache/nifi/security/xml/XmlUtilsTest.groovy

```diff
@@ -94,7 +94,7 @@ class XmlUtilsTest {
 
         // Assert
         logger.expected(msg)
-        assert msg =~ "SAXParseException.* DOCTYPE is disallowed when the feature"
+        assert msg =~ "SAXParseException.* DOCTYPE"
```

Comment:
This is the initial problem surfaced by user on Slack channel and in ticket.  The updated CI suggests that this change is sufficient to work in our test environments (EN, FR, JP).

https://issues.apache.org/jira/browse/NIFI-8478